File: C/0709.c
#include <string.h>

char* toLowerCase(char* s) {
    int len = strlen(s);

    for (int i = 0; i < len; i++) {
        if ('A' <= s[i] && s[i] <= 'Z') {
            s[i] = s[i] - 'A' + 'a';
        }
    }

    return s;
}

File: Python/0709.py
class Solution:
    def toLowerCase(self, s: str) -> str:
        s_list = []
        for i in range(len(s)):
            if (ord('A') <= ord(s[i])) and (ord(s[i]) <= ord('Z')):
                s_list.append(chr(ord('a') + ord(s[i]) - ord('A')))
            else:
                s_list.append(s[i])

        return "".join(s_list)
